test/test_ft_volumewrite.m

```diff
@@ -0,0 +1,16 @@
+function test_ft_volumewrite
+
+% WALLTIME 00:10:00
+% MEM 2gb
+% DEPENDENCY ft_volumewrite
+
+mrifilename   = dccnpath('/home/common/matlab/fieldtrip/template/headmodel/standard_mri.mat');
```

Comment:
could you use something like
```
[ftver, ftpath] = ft_version;
mrifilename = fullfile(ftpath, 'template', 'headmodel', 'standard_mri.mat');
```
here? That allows the script to run anywhere with fieldtrip installed, not only on our cluster.

Reply:
It's done. I think some of the previous functions have the same issue... I didn't check if you already modified them.